pages/en/cookbook/migrating-a-subgraph.mdx

```diff
@@ -4,18 +4,17 @@ title: Migrating an Existing Subgraph to The Graph Network
 
 ## Introduction
 
-This is a guide on how to migrate your subgraph from the Hosted Service to The Graph Network. The migration to The Graph Network has been successful for projects like Opyn, UMA, mStable, Audius, PoolTogether, Livepeer, RAI, Enzyme, DODO, Opyn, Pickle, and BadgerDAO all of which are relying on data served by Indexers on the network. There are now over 200 subgraphs live on The Graph Network, generating query fees and actively indexing web3 data.
+This is a guide on how to migrate your subgraph from the Hosted Service to The Graph's decentralized network. Migration to The Graph Network has been successful for projects like Opyn, UMA, mStable, Audius, PoolTogether, Livepeer, RAI, Enzyme, DODO, Opyn, Pickle, and BadgerDAO all of which are relying on data served by Indexers on the network. There are now over 200 subgraphs live on The Graph Network, generating query fees and actively indexing web3 data.
 
 The process of migration is quick and your subgraphs will forever benefit from the reliability and performance that you can only get on The Graph Network.
 
-### When not to Migrate?
+### Assumptions
 
-If your subgraph is:
+- You have already deployed a subgraph on the hosted service
+- The subgraph is indexing Ethereum mainnet
+- The subgraph does not have IPFS or full-text search dependencies (these are not fully supported on the decentralized network yet)
```

Comment:
Nitpicking, but these look like whole sentences, so maybe we should have periods at the end?
```suggestion
- You have already deployed a subgraph on the hosted service.
- The subgraph is indexing Ethereum mainnet.
- The subgraph does not have IPFS or full-text search dependencies (these are not fully supported on the decentralized network yet).
```